atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchManager.java

```diff
@@ -34,6 +34,10 @@
 
     // These methods are hidden on purpose as they should not be generally available, only for brave souls!
 
+    boolean isEnabled() {
+        return true;
+    }
```

Comment:
Why is this necessary? Also, if we're going to default to anything, wouldn't it be better to be false? I still think this should just be:
`boolean isEnabled;`

Reply:
I don’t think it matters, I just went with this approach.

Reply:
I still think we should not set this - I'd rather never, ever have to debug another bug like that auto-delegate, and while I know this is very different, it's just less error-prone to not set a default.

Reply:
(I know you have a test for it, but still).

Reply:
Done